descriptors/ModuleDescriptor-template.json

```
@@ -1,157 +1,167 @@
 {
-  "id": "${artifactId}-${version}",
-  "name": "Calendar module",
-  "requires": [],
+  "id": "@artifactId@-@version@",
+  "name": "Calendar for Service Point Opening Hours",
   "provides": [
     {
       "id": "calendar",
-      "version": "4.0",
+      "version": "5.0",
       "handlers": [
         {
-          "methods": [
-            "GET"
-          ],
-          "pathPattern": "/calendar/periods",
-          "permissionsRequired": [
-            "calendar.opening-hours.collection.get"
-          ]
+          "methods": ["GET"],
+          "pathPattern": "/calendar/calendars",
+          "permissionsRequired": ["calendar.endpoint.calendars.get"]
+        },
+        {
+          "methods": ["POST"],
+          "pathPattern": "/calendar/calendars",
+          "permissionsRequired": ["calendar.endpoint.calendars.post"]
```

There is a formatting convention for arrays in JSON in FOLIO that even for single entries they are formatted over multiple lines (like the previous entries).

```suggestion
          "permissionsRequired": [
            "calendar.endpoint.calendars.post"
          ]
```

---

Will fix after resolution of other permissions comments